src/pages/docs/projects/project-triggers/scheduled-deployment-trigger.md

```diff
@@ -57,3 +57,7 @@ The CRON expression must consist of all 6 fields, there is an optional 7th field
 | Month         | 1-12 or JAN-DEC      | * , - /                     | Y        |
 | Day of week   | 0-6 or SUN-SAT       | * , - / ? L #               | Y        |
 | Year          | 0001–9999            | * , - /                     | N        |
+
+:::div{.hint}
```

Comment:
I think this hint should be moved to the bottom of the Add a scheduled trigger section. It feels like it could be missed at the bottom of the cron section, and I think we want to make sure users are aware that this is the case even if they don't read the cron section.